product-service/handler.js

```diff
@@ -0,0 +1,60 @@
+const mock = require('./mock.json');
+// import mock from './mock.json';
+module.exports.getProductsList = async (event) => {
+  try {
+    const res = await Promise.resolve(mock);
+    console.log('res', res);
```

Comment:
If this is a development log, it's better to remove it before deploy, otherwise you will get this log in cloudwatch on every request